lib/wcc/contentful/store/base.rb

```diff
@@ -108,6 +108,14 @@ def apply(filter, context = nil)
       def op?(key)
         OPERATORS.include?(key.to_sym)
       end
+
+      def sys?(field)
+        field.to_s =~ /sys/
```

Comment:
What if I name my field one of the following:

* `sysadmin`
* `ecosystem`
* `systemic`
* `sysopsEmail`
* `photosynthesys` - b/c I misspelled it

Reply:
good call - regression test written and fixed